bin/input.txt

```diff
@@ -0,0 +1,17 @@
+todo borrow book
```

Comment:
You can .class files to the gitignore files so that complied files will be ignored